packages/salesforcedx-vscode-core/package.json

```
@@ -475,15 +475,15 @@
         },
         {
           "command": "sf.retrieve.current.source.file",
-          "when": "sf:project_opened && resourceLangId != 'forcesourcemanifest' && sf:in_package_directories && editorIsOpen && sf:has_target_org"
+          "when": "sf:project_opened && resourceLangId != 'forcesourcemanifest' && resourceLangId != 'yaml' && resourceLangId != 'als' && editorIsOpen && sf:has_target_org"
```

Command palette should match Editor View context menu.  We can keep `sf:in_package_directories` like we do there.

```suggestion
          "when": "sf:project_opened && resourceLangId != 'forcesourcemanifest' && sf:in_package_directories && editorIsOpen && sf:has_target_org"
```